zipkin-ui/test/component_data/spanNode.test.js

```diff
@@ -200,4 +200,27 @@ describe('SpanNodeBuilder', () => {
     expect(root.children.map(n => n.span))
       .to.eql([trace[3], trace[2], trace[1]]); // null first
   });
+
+  it('should order children by timestamp when IPs change ', () => {
```

Comment:
interestingly this passes which means I need to keep looking at that test trace! Maybe I missed the real culprit